tracer.go

```diff
@@ -116,7 +114,7 @@ type Tracer interface {
 
 // StartSpanOptions allows Tracer.StartSpan() callers and implementors a
 // mechanism to override the start timestamp, specify Span References, and make
-// Tags available at Span start time.
+// a single Tag or multiple Tags available at Span start time.
```

Comment:
I don't feel that we need Tags, but I don't feel strongly about removing it either. I'm generally not sure how we find consensus here (or how long to wait for comments)...